### PR TITLE
Python formatting tools

### PR DESCRIPTION
**Pull request by Xierumeng**

### CODE REVIEW

**Comment by Xierumeng** [comment]
DO NOT MERGE. Blocked by: https://github.com/UWARG/pathing/pull/36

**Comment by Xierumeng** [comment]
Mergeable.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    "Run",
+    "_",
+
+    # Return of main()
+    "result_main",
+]
+
+# Good variable names regexes, separated by a comma. If names match any regex,
+# they will always be accepted
+good-names-rgxs = "computer-vision-python"
```
Comment (TongguangZhang) [inline pyproject.toml:52]: remove

**Review by TongguangZhang — COMMENTED** [review]
Reviewed

**Review by TongguangZhang — APPROVED** [review]
LGTM!

**Pull request merged**